provider/win_driver_installer_test.py

```diff
@@ -525,3 +525,29 @@ def viofs_basic_io_test(test, params, vm):
         test.fail('The md5 value of host is not same to guest.')
     else:
         test.log.info("The md5 of host is as same as md5 of guest.")
+
+
+def balloon_test(test, params, vm, balloon_test_win):
+    """
+    Balloon service test in Windows guest.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param vm: The vm object
+    :param balloon_test_win: Basic functions for memory ballooning testcase
+    """
+    error_context.context("Running balloon service test...", LOG_JOB.info)
+    base_path = params.get("base_path", "/machine/peripheral/")
+    device = params.get("balloon", "balloon0")
+    device_path = base_path + device
+
+    session = vm.wait_for_login()
+    error_context.context("Config balloon service in guest", test.log.info)
+    balloon_test_win.configure_balloon_service(session)
+    balloon_test_win.get_memory_boundary("evict")
+    balloon_test_win.enable_polling(device_path)
+    res = balloon_test_win.get_memory_stat(device_path)
+    if "stats" not in str(res):
+        test.fail("The \"stats\" was NOT found.")
```

Comment:
Hi houqi ,could you check emory statistics during balloon instead of just checking if it is enabled?
You can refer to:

https://github.com/autotest/tp-qemu/blob/master/qemu/tests/balloon_service.py#L29